cmd/podman/varlink/io.podman.varlink

```diff
@@ -710,7 +711,7 @@ method WaitContainer(name: string, interval: int) -> (exitcode: int)
 #   "container": "62f4fd98cb57f529831e8f90610e54bba74bd6f02920ffb485e15376ed365c20"
 # }
 # ~~~
-method RemoveContainer(name: string, force: bool, removeVolumes: bool) -> (container: string)
+method RemoveContainer(name: string, force: int, removeVolumes: bool) -> (container: string)
```

Comment:
Please don't change the API. We need to maintain compatibility and this would break existing users. If we want to expose it to the varlink API, let's add another function.

@jwhonce PTAL

Reply:
@vrothberg @marcov I mentioned this in my review but also wanted to add comment.  Varlink is not designed to be versioned and to support changing the API.  New endpoints are their solution.